backend/inspirehep/hal/tasks.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+import structlog
+from celery import shared_task
+from inspire_utils.record import get_values_for_schema
+from sword2.exceptions import RequestTimeOut
+
+from inspirehep.hal.core.sword import create, update
+from inspirehep.hal.core.tei import convert_to_tei
+from inspirehep.records.api import LiteratureRecord
+
+LOGGER = structlog.getLogger()
+
+
+@shared_task(
+    bind=True,
+    retry_backoff=2,
+    retry_kwargs={"max_retries": 6},
+    autoretry_for=(RequestTimeOut,),
```

Comment:
So this goes to the general `celery` queue. We probably need a separate queue for this with separate workers, so we can tune the amount of concurrency if needed (I'll ask HAL if they have restrictions).

Reply:
I asked them and they have no limits currently. If we overwhelm them, they might need to put some in place. Still, I believe it's better to have a separate queue for this I believe to better see what's going on.